autoagents/eval/test_v3.py

```diff
@@ -0,0 +1,73 @@
+import asyncio
+import json
+
+from autoagents.agents.agents.search_v3 import ActionRunner
+from autoagents.agents.models.custom import CustomLLMV3
+
+
+async def work(user_input):
+    outputq = asyncio.Queue()
+    llm = CustomLLMV3(model_name="action_vicuna_7b_withabort_v2_16k", max_tokens=1024)
```

Comment:
1024 doesnt seem like a lot for `max_tokens`

Reply:
The arguments are from an arbitrary example used for functional testing of code. The end user should change as per the allowed model context size etc.